oscilloscope/agilent.py

```diff
@@ -0,0 +1,195 @@
+"""
+Wrappers for Agilent oscilloscopes.
+
+"""
+
+
+from __future__ import absolute_import
+import oscilloscope.wrappers as wrappers
+import labrad.units as U
+import struct
+import numpy as np
+from twisted.internet.defer import inlineCallbacks, returnValue
```

Comment:
I'm really glad to know I'm not the only person who twitches when imports aren't organized.

Fixed.